HISTORY.md

```diff
@@ -1,3 +1,8 @@
+# v2.3.0
```

Comment:
Changes to the history will be made as part of the release process. Please remove this change.